xla/service/gpu/runtime/nccl_collective_thunk.cc

```diff
@@ -244,7 +244,7 @@ static absl::StatusOr<NcclCliqueKey> GetNcclCliqueKey(
                        stream_kind);
 }
 
-absl::StatusOr<NcclApi::NcclCommHandle> GetNcclComm(
+absl::StatusOr<std::pair<NcclApi::NcclCommHandle, bool>> GetNcclComm(
```

Comment:
prefer struct to a std::pair, the meaning of a bool is hard to guess without reading implementation

Reply:
updated